src/calculateRentalCost.js

```diff
@@ -29,6 +29,17 @@
 
 function calculateRentalCost(days) {
   // write code here
+  let result = 0;
+
+  if (days >= 3 && days < 7) {
+    result = days * 40 - 20;
```

Comment:
2. [checklist] - Don't use literals in expressions (HARDCODE). Instead use constants explaining this values